style.css

```diff
@@ -0,0 +1,1165 @@
+    font-size: 0.8em; }
+
+
+.box5-right a{
+    text-decoration: none;
+    color: black; }
+
+
+.box5-right a:hover{ color: orangered;}
+
+@media screen and (min-width: 768px) and (max-width: 1024px){
+    .fifth{    margin-top: 8%;
+    }
+}
+
+/* 6 - Sixth container - Latest news */
+
+.box6{
+    height: 7.4rem;
+    border-top: 1px solid gainsboro;
+    margin: 2%; }
+
+#first-latest{
+    border: none;
+}
+
+.box6-left{
+    width: 40%;
+}
+
+.box6-left img{
+    padding-top: 20%;
+    max-width: 100%;
+    max-height: 100%;
+}
+
+.box6-right{
+    width: 60%;
+    padding-top: 8%;
+    padding-left: 3%;
+}
+
+.box6-right h2{
+    font-size: 1em;
+    font-weight: bold;
+}
+
+.box6-right h2:hover{
+    color: orangered;
+}
+
+
+@media screen and (max-width:768px){
+    .box6{height: 9.4rem;}
+}
+
+
+/* 7 - Seventh container - Sign up */
+
+
+.box7{
+    
+    height: 11rem;
+    width: 18rem;
+    background:whitesmoke;
+    border: 1px solid gainsboro;
+    
+
+}
+
+.box7-up{
+    height: 40%;
+}
+
+.box7-up-left{
+
+    height: 100%;
+    width: 25%;
+    padding-top: 5%;
+
+}
+
+.box7-up-left img{
+    width: 40px;
+    height: 40px;}
+
+
+.box7-up-right{
+
+    height: 100%;
+    width: 75%;
+    padding-top: 5%;
+
+
+}
+.box7-up-right h2{
+    font-weight: bold;
+    font-size: 1em;
+}
+
+.box7-mid{
+    height: 40%;
+    padding-top: 5%;
+    padding-left: 5%;
+    
+}
+
+.btn7{
+    height: 70%;
+    width: 35%;
+    padding-top: 4%;
+    background: dimgray;
+}
+
+
+.box7-down{
+    height: 20%;
+    padding-left: 5%;
+}
+
+.box7-down h2{font-size: 0.9em; font-weight: bold; color: dimgray;}
+
+.btn7:hover{background: rgb(179, 179, 179);}
+
+.btn7 h2{
+    font-size: 1em;
+    font-weight: bold;
+    color: white;
+}
+
+@media screen and (max-width: 768px){
+    .box7{    margin-top: 15%;
+              margin-left: 5%;}
+}
+
+@media screen and (min-width: 768px){
+    .box7{    margin-top: 15%;
+              margin-left: 5%;}
+}
+
+/* 8 - eight container - See all features */
+
+
+.box8{
+
+    height: 120vw;
+}
+
+.tags{background: white; width: 28%; opacity: 0.8;
+    height: 7%;
+    color: rgb(255, 17, 0); font-weight: bold;
+   font-size: 0.9em;
+   position: absolute;    
+   bottom: 0;
+   left: 0;
+   padding-top: 2%;
+   padding-left: 1%;
+   }
+
+.box8-up{
+    height: 90%;
+    width: 100%;
+    background-image:url(images/box8.jpg);
+    background-size: cover;
+    background-position: center center;
+    min-height: 500px;
+    min-width: 350px;
+    position: relative;
+}
+
+.box8-down{
+    height: 10%;
+    width: 100%;
+    background: whitesmoke;
+    padding-top: 1%;
+}
+
+.box8-down h2{
+    font-size: 1em;
+    font-weight: bold;
+    color: black;
+}
+
+
+
+
+
+@media screen and (max-width:768px){
+    .eight{margin-top: 130%;}
+
+}
+/* 9 - ninth container - in the magazine */
+.ninth{margin-bottom: 6%;}
+
+.box8-big-up{
+    height: 90%;
+    width: 100%;
+    background-image:url(images/box8.jpg);
```

Comment:
Opinion/Tip (not required)
When you have similar background properties like this in one class definition it's also a good candidate for `background` property.

Same with the font. If you agree you are free to identify cases like these in your code and implement changes accordingly. If you disagree, that's fine too. It's a matter of opinion.

Reply:
thanks